project/tests_codeception/Acceptance/Test15_StorageSeeCest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+
+namespace TestsCodeception\Acceptance;
+
+use TestsCodeception\Support\AcceptanceTester;
+
+class Test15_StorageSeeCest
+{
+
+    public function tryToTest(AcceptanceTester $I)
+    {
+        $I->wantTo('Check whether manager can check storage');
+
+        $I->amOnPage("/");
+        $I->fillField("email", "kierownik@email.com");
+        $I->fillField("password", "kierownik");
+        $I->click("Zaloguj się");
+
+        $I->seeCurrentUrlEquals("/");
+        $I->see("Magazyn");
+        $I->click("Magazyn");
+        $I->seeCurrentUrlEquals("/storage");
+        $I->see("NAZWA");
+        $I->see("ILOŚĆ");
+        $I->see("Strzykawka");
+        $I->see("Łóżko");
+        $I->dontSee("Butelka");
```

Comment:
Tutaj powinien być jakiś wybór placówki?

Reply:
Z tego co wiem to w testach akceptacyjnych nie da się sprawdzić wyszukiwarki, więc zrobiłam dla pierwszej placówki.

Reply:
Działanie wyszukiwarki jest testowane osobno - manualnie, ponieważ w testach akceptacyjnych wyszukiwarka jest niereaktywna.